view/utils/init-table-sorting-on-client.js

```diff
@@ -16,6 +16,23 @@ module.exports = function (domElement/*, opts */) {
 		// set type, either numeric or text 
 		type: 'numeric'
 	});
+	window.jQuery.tablesorter.addParser({
+		// set a unique id 
+		id: 'dates',
+		is: function (s) {
+			// return false so this parser is not auto detected 
```

Comment:
@mtuchowski not sure about internals, but to my understanding this assures that we will not use it by accident (which inmo is undesired).